src/client/Microsoft.Identity.Client/Platforms/Android/AndroidBroker.cs

```diff
@@ -96,7 +96,6 @@ private async Task AcquireTokenInternalAsync(IDictionary<string, string> brokerP
                     _logger.Verbose("User is specified for silent token request. Starting silent broker request");
                     string silentResult = await _brokerHelper.GetBrokerAuthTokenSilentlyAsync(brokerPayload, _activity).ConfigureAwait(false);
                     s_androidBrokerTokenResponse = CreateMsalTokenResponseFromResult(silentResult);
```

Comment:
I don't think this covers the case where the broker returns an error result.

Reply:
The error case is being handled by the CreateMsalTokenResponseFromResult method. we check to see if we get an account back from the account manager and if we dont we log an error. however, it wouldnt hurt to also add that error to the MSAL token response.